mealie/core/security/jwt_validation.py

```diff
@@ -0,0 +1,19 @@
+import json
```

Comment:
`jwt_validation` contains the logic to decoded and extract the information of the JWT using JWKS.